.github/workflows/validate-crd-docs.yml

```diff
@@ -0,0 +1,38 @@
+name: Validate CRD Docs
+on:
+  pull_request:
+    branches:
+      - 'main'
+defaults:
+  run:
+    shell: bash
+jobs:
+  check-helm-docs:
+    name: Check crd documentation values
+    runs-on: ubuntu-22.04
```

Comment:
Minor - other workflows use ubuntu-latest [1]. It might be best to use the same :) 

[1] - https://github.com/open-feature/open-feature-operator/blob/main/.github/workflows/release-please.yml#L23